tools/cpm_tpm_rpk/cpm_tpm_rpk.xml

```diff
@@ -0,0 +1,100 @@
+<tool id="cpm_tpm_rpk" name="CPM, TPM, RPK" version="0.1.0">
```

Comment:
name="Generate CPM, TPM and RPK"
<description>from raw count expression values</description>